packager/live_packager_test.cc

```diff
@@ -292,6 +314,46 @@ void CheckVideoInitSegment(const FullSegmentBuffer& buffer,
   }
 }
 
+void CheckTextInitSegment(const FullSegmentBuffer& buffer,
+                          media::FourCC handler,
+                          media::FourCC format) {
+  bool err(true);
```

Comment:
nit: could we follow one style for initialization,  I see that overall it's  `type var = value`

Reply:
This seems to be consistent with the style of the codebase.  There are places in the codebase that also use this style of initialization, e.g., https://github.com/Pluto-tv/shaka-packager-live/blob/pluto-cmake/packager/media/formats/mp4/mp4_media_parser.cc#L255